api-v3-stable.rst

```diff
@@ -0,0 +1,48 @@
+.. post:: Feb 1, 2021
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+API v3 is now stable!
+=====================
+
+We are excited to announce that our API v3 has reached its stable version,
+and it's now available for all Read the Docs users.
+This includes Read the Docs Community and Read the Docs for Business.
+Since we :doc:`announced the beta version </api-v3-beta>` of it,
+we have been adding extra functionality and bug-fixing some minor issues based on users feedback.
+
+The new API v3 *is not* a fully replacement (yet!) of API v2,
+but **we highly recommend using API v3 for all the new code you write from now on**.
+API v2 is planned to be deprecated soon,
+though we have not yet set a time frame for deprecation.
+We will alert users with our plans when we do.
+
+
+Nice features
+-------------
+
+These are some of the possible actions API v3 allows and we like the most,
+
+- Ability to import new projects
+- Activate a version
+- Trigger builds
+- Check build status
+- Manage redirects
+
+They allow you to easily manage common tasks like setup the documentation for a new project,
```

Comment:
What is `They`?

```suggestion
API v3 allow you to easily manage common tasks like setting up documentation for a new project,
```

Reply:
"They" refers to the features mentioned above :sweat_smile: